src/App/ImagePlane.cpp

```diff
@@ -25,19 +25,36 @@
 #include "ImagePlane.h"
 
 
-using namespace Image;
 using namespace App;
 
-PROPERTY_SOURCE(Image::ImagePlane, App::GeoFeature)
+PROPERTY_SOURCE(App::ImagePlane, App::GeoFeature)
 
 
 ImagePlane::ImagePlane()
+    : XPixelsPerMeter{1000.0}

```

Comment:
Where is this 1000 coming from? It feels sort of "magic-number-y".

Reply:
Qt says "dots per meter" and for most of the tested images this value was 3750 or 3850.  Now I needed a sensible default value and because further down in the code the XPixelsPerMeter is divided by 1000 to get the size in mm it looked like a not totally wrong choice.

So, if for any reason the value couldn't be determined than we use `1000/1000` to convert px into mm.

But if you know how to get the "dots per meter" independent of the Qt API we can use that instead.

Reply:
Nothing to do here right now. So I mark it as resolved.